### PR TITLE
Leave a hint in step-7.

### PR DESCRIPTION
**Pull request by bangerth**
This has apparently been sitting in my github for a year now...

### CODE REVIEW

File: examples/step-7/doc/results.dox [hunk 1]
```diff
@@ -288,10 +288,11 @@ work than if we used global refinement. This is not a particularly
...
-plotted run time against the number of unknowns by timing each
-refinement step (e.g., using the Timer class), you will notice that
-the linear solver is not perfect -- its run time grows faster than
-proportional to the linear system size -- and picking a better
-linear solver might be appropriate for this kind of comparison.
+Of course, a fairer comparison would be to plot runtime (switch to
+release mode first!) instead of number of unknowns on the $x$ axis. If
+you plotted run time (check out the Timer class!) against the number
+of unknowns by timing each refinement step (e.g., using the Timer
+class), you will notice that the linear solver is not perfect -- its
```
Comment (kronbichler) [inline examples/step-7/doc/results.dox:295]:
```suggestion
of unknowns by timing each refinement step, 
you will notice that the linear solver is not perfect -- its
```

**Review by kronbichler — APPROVED** [review]
I have a small suggestion.

Regarding the last comment about an solver, I wonder if we should give more explicit hints. Among the many works which have done this, one example pretty close to the step-7 tutorial program, is Figure 5 of https://doi.org/10.1137/16M110455X, including a comparison of different polynomial degrees and different kinds of solvers (matrix-free, matrix-based, continuous FEM, DG, HDG). That is my own paper, so I feel conflicted to suggest including that link (also, it is not open-access), but I think a reader might want to read what we consider state-of-the-art.

**Comment by bangerth** [comment]
Excellent idea. I added the reference, I think it's quite useful!

**Comment by bangerth** [comment]
Ping?

**Comment by bangerth** [comment]
Could I interest someone to hit the button on this one?

**Review by drwells — APPROVED** [review]

**Pull request merged**